File: Lucky Ticket/lucky-ticket.js
/**
 * Check is lucky ticket number
 *
 * @param {int} n
 * @return {boolean}
 */
const isLucky = n => {
  const arr = (""+n).split('').map(v => {
    return parseInt(v);
  });
  const middle = parseInt(arr.length / 2);

  return arr.slice(0, middle).reduce((a, b) => a + b, 0) === arr.slice(middle).reduce((a, b) => a + b, 0);
};

console.log(isLucky(1230)); // true
console.log(isLucky(239017)); // false

File: Lucky Ticket/lucky-ticket.py
def isLucky(n):
    """
    :type n: int 
    :rtype: bool 
    """
    arr = [int(x) for x in str(n)]
    middle = int(len(arr) / 2)

    return sum(arr[0:middle]) == sum(arr[middle:])


print(isLucky(1230))  # True
print(isLucky(239017))  # False